frontend/components/Navbar/Export.tsx

```diff
@@ -1,14 +1,14 @@
 import api from "@api/Api";
 
-export default function Export() {
+export default function Export({ 'data-testid': dataTestId }: { 'data-testid'?: string }) {
```

Comment:
Just curious why move the test id as component prop?

Reply:
More abstraction basically. But feel free to update it. Would you prefer another approach?